### PR TITLE
credit card: use the Luhn algorithm to validate the number

### PR DESCRIPTION
**Pull request by goneri**

### CODE REVIEW

**Comment by goneri** [comment]
@justjais can you review this please?

File: anonymizor/anonymizor.py [hunk 1]
```diff
@@ -123,8 +123,15 @@ def is_valid_telephone_number(value: str) -> bool:
 
 
 def is_valid_credit_card_number(value: str) -> bool:
+    def luhn(n):
```
Comment (justjais) [inline anonymizor/anonymizor.py:126]: @goneri Do we need the Luhn algorithm for CC detection, as we're concerned here with generic 16-digit CC numbers and the relevance of the CC number being valid or invalid is not much of a concern?

Comment (goneri) [reply anonymizor/anonymizor.py:126]: My concern is more that we identify any random string with 16 characters as a CC number and we start to destroy them all. This being said, it's not a common pattern, so the impact would be limited.

**Comment by goneri** [comment]
Closing, the change is also part of https://github.com/ansible/ansible-wisdom-anonymizor/pull/3

**Pull request closed without merging**